src/Hl7.Fhir.Support.Poco/Support/Issue.cs

```diff
@@ -42,7 +42,12 @@ public OperationOutcome.IssueComponent ToIssueComponent(string message, string p
             // Put numeric code + readable message into a CodeableConcept
             ic.Details = ToCodeableConcept(message);
 
-            if (path != null) ic.Location = new List<string> { path };
+            if (path is not null)
+            {
+                ic.Expression = new List<string> { path };
+                // IssueComponent.Location is deprecated but we still set this because of backwards compatibility.
```

Comment:
Or should we change OperationOutcome.cs in common so that if you set Location, you set Expression and vice versa?

Reply:
Just a tip that I've continued to use the location property for other error location styles (file positions line col, or questionnaire linkID, and having the fhirpath expression with the real one...)

Reply:
But I'm also happy for this to be copying the data in both properties here (as the location field was set by this code previously, so all should be fine)

Reply:
> Just a tip that I've continued to use the location property for other error location styles
I was kinda expecting that yes, connecting the two together at the property level is also not backwards compatible...

So I found code setting Location in the STU3 PR (https://github.com/FirelyTeam/firely-net-sdk/pull/2048) where we then should set both Location and Expression (instead of making the setters for both the same).